.github/workflows/lint.yaml

```diff
@@ -0,0 +1,27 @@
+name: Lint
+
+on: [ push, pull_request ]
+
+jobs:
+  lint:
+    name: Lint
```

Comment:
Lets split this into two? It'd be nice to require both that it builds and that it lints but I feel they are two distinct things and don't actually rely on one another or need to be ran in series.

Reply:
Done I seperated them into 2 different jobs. Or do you prefer that I split them into two files `lint.yaml` and `build.yaml`?

Reply:
Ah, yes I hadn't realized the file name. 

Two separate files would be more appropriate than one called `lint.yaml`. Perhaps even better may be to change `lint.yaml` to `validate.yaml` and keep the 2 jobs inside the one "validation workflow". 

It seems the intent here is to "validate" the projects styling and that it can be built.

Reply:
Ok done I renamed the file.